pom.xml

```diff
@@ -1562,7 +1567,6 @@ under the License.
 				<configuration>
 					<!-- enables TCP/IP communication between surefire and forked JVM-->
 					<forkNode implementation="org.apache.maven.plugin.surefire.extensions.SurefireForkNodeFactory"/>
-					<forkCount>${flink.forkCount}</forkCount>
```

Comment:
there are a number of modules that override the forkCount to 1 using the general forkCount option (e.g., various connector modules), which afaik would now always ignored because the execution-specific option takes precedence.

I don't know whether that's a problem though. I haven't seen an increase in test instabilities when I used these commits in a WIP branch, but those also didn't ran on Azure.
In any case we either need to remove these now-ignored forkCount settings, or adjust the poms to override it in the execution instead (or modify the properties; I think that works as well).
I would be inclined to just remove them and see what happens.

huh...there should also be some forkReuse settings that are now ignored after fee8217a65bdbcf2eed8b2c8a31852f84f1022ad. Might wanna get rid of those as well...

Reply:
I'm looking into preparing a cleanup PR that removes reuseForks, other custom forkCounts etc.
Then we should also get a clearer picture of how much this chain specifically gets us.